compiler/fir/fir2ir/src/org/jetbrains/kotlin/fir/backend/generators/CallAndReferenceGenerator.kt

```diff
@@ -87,12 +98,18 @@ class CallAndReferenceGenerator(
                 }
                 is IrFunctionSymbol -> {
                     val function = symbol.owner
-                    IrFunctionReferenceImpl(
-                        startOffset, endOffset, type, symbol,
-                        typeArgumentsCount = function.typeParameters.size,
-                        valueArgumentsCount = function.valueParameters.size,
-                        reflectionTarget = symbol
-                    )
+                    // TODO: should refer to LanguageVersionSettings.SuspendConversion
```

Comment:
@mglukhikh I think both resolution and conversion should refer to feature flags, but didn't so far. Any recommended place to add it? E.g., `FirSession` or some other context?